docs/content/en/docs/pipeline-stages/testers/structure.md

```diff
@@ -22,16 +21,16 @@ If frequent tests are prohibitive, long-running tests should be moved to a dedic
 This following example shows the `test` section from a `skaffold.yaml`.
 It instructs Skaffold to run all container structure tests in the `structure-test` folder relative to the Skaffold root directory:
 
-{{% readfile file="samples/testers/test.yaml" %}}
```

Comment:
```
Structure tests are defined per image in the Skaffold config.
Every time an artifact is rebuilt, Skaffold runs the associated structure tests on that image.
If the tests fail, Skaffold will not continue on to the deploy stage.
```

The custom test section should also mention these things. 

```
If frequent tests are prohibitive, long-running tests should be moved to a dedicated Skaffold profile.
```
Mention that they can be turned off using the `skipTests` flag here also

Reply:
Updated the content.